_layouts/projects.html

```diff
@@ -6,68 +6,76 @@
 		<main id="main" role="main">
 			<section class="project id="">
 				<header class="project--header">
+					<p class="project--tags">
+						<span class="project--type">{{ page.type-of-product | default: "N/A"}}</span> ∙ {{ page.tags }}
+					</p>
 					<h1 class="project--title">{{ page.title }}</h1>
-					<h3 class="project--source-link">
-						<a href="{{ page.link-to-source }}"
+					<div class="project--link">
+						<span class="project--link-tag">Repository</span>
+						<a href="{{ page.link-to-source }}" class="project--url"
 							>{{ page.link-to-source }}</a
 						>
-					</h3>
+					</div>
+					{% if page.link-to-live-product %}
+					<div class="project--link">
+						<span class="project--link-tag">Live Site</span>
+						<a href="{{ page.link-to-live-product }}" class="project--url"
+							>{{ page.link-to-live-product }}</a
+						>
+					</div>
+					{% endif %} 
 				</header>
 				<h2>Summary</h2>
-				<p>{{ page.summary }}</p>
-				<p>
-					<span class="project--label">Type of product: </span>{{
-					page.type-of-product }}
-				</p>
-				<p>
-					<span class="project--label">Platform: </span>{{
-					page.platform }}
-				</p>
-				<p>
-					<span class="project--label">Initial audience: </span>{{
-					page.initial-audience }}
-				</p>
-				<p>
-					<span class="project--label">Author: </span>{{
-					page.organizing-org }}
-				</p>
+				<p class="project--summary">{{ page.summary }}</p>
+				<div class="project--details">
+					<div class="detail-column">
+						<p>
+							<span class="project--label">Platform </span>{{
+							page.platform }}
+						</p>
+						<p>
+							<span class="project--label">Initial audience </span>{{
+							page.initial-audience }}
+						</p>
+						<p>
+							<span class="project--label">Author </span>{{
+							page.organizing-org }}
+						</p>
```

Comment:
Would this break if we wrapped it in an `{% if %}` block? If so, we can make a `| default: "N/A"` here